nginx-aspnet-mysql/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+services:
+  backend:
+    build: backend
+    secrets:
+      - db-password
+    depends_on: 
+      - db
+    environment:
+      - ASPNETCORE_URLS=http://+:8000
+  
+  db:
```

Comment:
Would you mind adding a `healthcheck` please? The db takes some time to start and the backend may fail because of that. 
```yaml
services:
  backend:
    ...
    depends_on:
      db:
        condition: service_healthy
  db:
    ...
    healthcheck:
      test: ["CMD", "mysqladmin", "ping", "-h", "127.0.0.1", "--silent"]
      interval: 3s
      retries: 5
      start_period: 30s
```
We need to update samples and add healthchecks everywhere.

Reply:
Sure. Done! Please approve @aiordache